### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by SuperQ**
Add a github issue template to direct help requests to other channels.

### CODE REVIEW

**Comment by brian-brazil** [comment]
We should probably ask for the type of device they're using for support/bugs, so we can build of a blacklist of what's broken.

**Comment by SuperQ** [comment]
Good idea, I added that and a question about where to find the MIB(s).

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!--
+	Please note: GitHub issues should only be used for feature requests and
+	bug reports. For general discussions, please refer to one of:
```
Comment (brian-brazil) [inline .github/ISSUE_TEMPLATE.md]: discussions and support

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**